src/app/pages/quickMaps/pages/projection/projectionFoodSources/projectionFoodSources.component.ts

```diff
@@ -277,11 +293,26 @@ export class ProjectionFoodSourcesComponent implements AfterViewInit {
               stacked: true,
               scaleLabel: {
                 display: true,
-                labelString: 'value',
+                labelString: this.micronutrientName + ' in mg/capita/day',
```

Comment:
Not sure if it applies if all values will be in mg, but you can call out to the quickMapsService and return the respective micronutrient unit defined by the mineral or vitamin. 
`
        this.quickMapsService.micronutrientObs.subscribe((micronutrient: MicronutrientDictionaryItem) => {
          this.mnUnit = null == micronutrient ? '' : micronutrient.unit;
        }),
`

Reply:
good shout, added in commit https://github.com/micronutrientsupport/micronutrient-support-tool/pull/498/commits/e79d627c5d67e5b349a849ef285ddb9299a4945e

Reply:
@DanielLyleWilliamWarren ready for re-review